contrib/utils.cpp

```diff
@@ -100,6 +100,27 @@ BIGINT next235even(BIGINT n)
   return nplus;
 }
 
+BIGINT next235beven(BIGINT n, BIGINT b)
```

Comment:
Can `next235even` be written to call this function?

Reply:
Yes, they can (see 5f75d96).

Reply:
Great. Thank you.